pkg/client/unversioned/daemon_sets.go

```diff
@@ -35,7 +36,7 @@ type DaemonSetInterface interface {
 	Update(ctrl *extensions.DaemonSet) (*extensions.DaemonSet, error)
 	UpdateStatus(ctrl *extensions.DaemonSet) (*extensions.DaemonSet, error)
 	Delete(name string) error
-	Watch(label labels.Selector, field fields.Selector, resourceVersion string) (watch.Interface, error)
+	Watch(label labels.Selector, field fields.Selector, opts api.ListOptions) (watch.Interface, error)
```

Comment:
We should change that so that we only pass "opts" to both "List()" and "Watch()" function - I would like to do it, but in a separate PR. FYI: @lavalamp @smarterclayton